packages/mini-browser/src/common/mini-browser-service.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+export const MiniBrowserServicePath = '/services/mini-browser-service';
+export const MiniBrowserService = Symbol('MiniBrowserService');
+export interface MiniBrowserService {
+
+    /**
+     * Resolves to an array of file extensions supported by the `Mini Browser`.
+     *
+     * The extensions start with the leading dot (`.`) and should be treated in a case-insensitive way. This means,
+     * if the `Mini Browser` supports `['.jpg']`, then it can open the `MyPicture.JPG` file.
+     */
+    supportedFileExtension(): Promise<string[]>;
```

Comment:
According to the return type, the name should be plural (`supportedFileExtensions`)?

Reply:
👍 Nice catch. I have fixed it.